system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -1761,7 +1761,7 @@ class CLdapClient : implements ILdapClient, public CInterface
         }
         else if(rtype == RT_WORKUNIT_SCOPE)
         {
-            int defaultWorkunitScopePermission = -2;
+            SecAccessFlags defaultWorkunitScopePermission = (SecAccessFlags)-2;
```

Comment:
-2 is a value that is unknown to the enumeration. It is used to signal this code that we were not able to query the defaultWorkunitScopePermission and therefore not to use it.  I will add comments explaining